.github/workflows/buf-ci.yaml

```
@@ -2,15 +2,17 @@ name: Buf CI
 on:
   push:
     paths:
-      - '**.proto'
-      - '**/buf.yaml'
-      - '**/buf.lock'
-      - '**/buf.md'
-      - '**/README.md'
-      - '**/LICENSE'
-      - .github/workflows/buf-ci.yaml
+      - 'proto/**.proto'
+      - 'proto/**/buf.yaml'
+      - 'proto/**/buf.lock'
```

```suggestion
      - 'buf.yaml'
      - 'buf.lock'
```
Safe to presume this action only runs for the input defined at the root.